lib/barbeque_client/executor.rb

```diff
@@ -8,9 +8,19 @@ class Executor
     # @param [String] queue_name - barbeque's job_queues.name
     def initialize(job:, message:, message_id:, queue_name:)
       @job        = job
-      @message    = JSON.load(message)
```

Comment:
`Array(arg)` converts arg into an Array implicitly (`Array(foo: :bar) # => [[:foo, :bar]]`), and `Array.wrap` is not Ruby's method but ActiveSupport's, then I don't want introduce AS this time...